metricbeat/docker-compose.yml

```diff
@@ -17,11 +17,11 @@ services:
 
   # Used by base tests
   elasticsearch:
-    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-8.0.0}-1
+    image: docker.elastic.co/integrations-ci/beats-elasticsearch:${ELASTICSEARCH_VERSION:-8.1.0}-1
     build:
       context: ./module/elasticsearch/_meta
```

Comment:
@matschaffer Unrelated to this PR but stumbled over this line here. This explains why module and this testing setup are kind of related.